lib/pg_eventstore/errors.rb

```diff
@@ -137,7 +137,7 @@ def initialize(set, name, lock_id)
 
     # @return [String]
     def user_friendly_message
-      <<~TEXT
+      <<~TEXT.strip
         Could not lock Subscription from #{set.inspect} set with #{name.inspect} name. It is already locked by \
```

Comment:
```suggestion
        Could not lock subscription from #{set.inspect} set with #{name.inspect} name. It is already locked by \
```

I would generally use lower case for words that are not names.